### PR TITLE
Update CI

### PR DESCRIPTION
**Pull request by nim65s**
Hi,

This PR:
- Update the docker image generation documentation and Dockerfiles
- Remove latex & gs, to let doxygen use MathJax to be faster
- Allow the CI to be ran on other namespaces than `humanoid-path-planner`
- Ensure that the Makefile stops on errors

### CODE REVIEW

**Comment by nim65s** [comment]
This leads to the pipeline : https://gepgitlab.laas.fr/gsaurel/hpp-doc/pipelines/1432

File: doc/Makefile [hunk 1]
```diff
@@ -199,7 +199,7 @@ all: hpp_tutorial.install hpp-gepetto-viewer.install
 test-ci: baxter_common.install romeo.install universal_robot.install \
 	hpp-environments.install
 	${MAKE} hpp_tutorial.install hpp-gepetto-viewer.install \
-	hpp_universal_robot.install; \
+	hpp_universal_robot.install && \
```
Comment (jmirabel) [inline doc/Makefile]: Does this make *make* behave differently ?

**Comment by jmirabel** [comment]
I did not check carefully but it seems good to me.

File: doc/Makefile (hunk 1 above)
Comment (nim65s) [reply doc/Makefile]: Yes, if the first command fails, the second is executed anyway, it append a few times to me before I could understand the issue…

**Pull request merged**